### ISSUE TITLE
Improve the Aesthetic and Cohesion of Download Button in the Title Bar

### ISSUE THREAD

**Issue by DanVanAtta** [triplea-game/triplea-game.github.io#5]
The 'download' button in the menu bar I think is the same button from the home page. Those two buttons being similar is a very good thing, but it does seem to me like that button is not quite right for the menu bar. Perhaps just making it a bit smaller would help.

My issue is that it does not look very cohesive, it does not look like the red download button belongs, or is even really a part of the menu bar.

We could potentially make the button look exactly like another menu bar link as another alternative suggestion. I'd start probably with trying to make the button a bit smaller maybe. I am also curious if there are any other ideas out there, and if this is also bothersome to other people as well.

**Comment by williamdekryger** [comment]
I agree this could be an issue. I will play with different styles this weekend, unless you had something specifi in mind (e.g. live on another site).

**Comment by williamdekryger** [comment]
How's this for a fix?

![smallerdlbutton](https://cloud.githubusercontent.com/assets/5250014/10397936/c75d9e70-6e99-11e5-8c8f-6904cd4f61b7.png)

**Comment by DanVanAtta** [comment]
Interesting. I think now the buttons being different makes it seem like they might go to different places. It's something I had not expected. Perhaps that is something I need to just get over and it's just me. 

I think it being off to the right is a pretty strong callout. I'm curious what would happen if it had the same font and color scheme as the rest of the header? I think that might be the way to go *maybe*. 

bottom line, the smaller size is probably better, should look a good bit better on the other pages. Side by side to the redundant-by-design button, less so.

**Comment by williamdekryger** [comment]
Yeah, I always use the screenshots that make it look the most redundant, unfortunately. Frostion from the dev forum has frequently called me out on that *sigh* I may need to see if Jekyll supports having a homepage body class, so I can hide that button on the homepage.

**Comment by DanVanAtta** [comment]
I wouldn't have the header button appear and reappear. Simply make it another header link? I'm quickly becoming more and more convinced that is the right way to go. We will have a dedicated download page, so it makes sense to have a header link to get there from any other page.

**Comment by DanVanAtta** [comment]
With the button recently made smaller it does look better.
